accord-core/src/main/java/accord/local/SafeCommandStore.java

```diff
@@ -155,6 +156,7 @@ <T> T mapReduce(Seekables<?, ?> keys, Ranges slice,
     NodeTimeService time();
     CommandStores.RangesForEpoch ranges();
     Timestamp maxConflict(Seekables<?, ?> keys, Ranges slice);
+    void registerHistoricalTransactions(Deps deps);
```

Comment:
I had intended to keep this to `CommandStore` to isolate the APIs - `SafeCommandStore` is for working with `Command` and `SafeCommand` whereas `CommandStore` is for internal book-keeping operations.

Reply:
sorry, just saw this after merge...

In order to make the outcome of this method durable, it has to be in the safe store for C*, so felt it was best to move it there as that's were we mutate state normally